.github/workflows/build.yml

```diff
@@ -48,7 +48,7 @@ jobs:
           name: artifacts
           path: build/artifacts/
 
-      - name: Zip temp logs
+      - name: Collect integration test logs
```

Comment:
FYI, they're collecting the logs from the Configure Visual Studio step as well.

Reply:
That is true (plus any other unrelated *.log files). I've only ever used the log files to debug burn integration test files. The name `test_burn_%GITHUB_RUN_ID%.zip` suggests that is its primary purpose. Would it be clearer if the name of the .zip file was changed? Given I've only used it for debugging burn integration tests, it makes sense to me right now but I'd be interested if other names are more appropriate.

Reply:
Probably. I'm still annoyed at having to extract a zip file from a zip file, but I learned that that is the best way to upload a bunch of files with Github Actions right now.